Source/EasyNetQ.Tests/PublishExchangeDeclareStrategyExtensions.cs

```diff
@@ -1,8 +1,9 @@
 ﻿using System;
 using System.Threading;
+using EasyNetQ.Producer;
 using EasyNetQ.Topology;
 
-namespace EasyNetQ.Producer
+namespace EasyNetQ.Tests
```

Comment:
Is this file used only in tests?

Reply:
Because they are needed only in tests currently.